extras/check_forbidden_types.sh

```diff
@@ -0,0 +1,34 @@
+#! /bin/bash
```

Comment:
```suggestion
#!/usr/bin/env bash
```

Reply:
The NEST hackathon decided to go for consistency with the other files in the repository here and just remove the space:
```suggestion
#!/bin/bash
```